stylix/testbed/themes/cursorless.nix

```diff
@@ -8,6 +12,6 @@ in
     enable = true;
     image = images.dark;
     base16Scheme = "${tinted-schemes}/base16/catppuccin-macchiato.yaml";
-    polarity = "dark";
+    themeGeneration.polarity = "dark";
```

Comment:
Again, does this have the relevant aliases?

Reply:
there's a warning that the attribute name changes

Reply:
I can't see a `lib.mkRenamedOptionModule` or a `lib.warn` call anywhere. Where should I be looking?